go.mod

```diff
@@ -32,6 +32,7 @@ require (
 	github.com/influxdata/flux v0.169.0
 	github.com/influxdata/httprouter v1.3.1-0.20191122104820-ee83e2772f69
 	github.com/influxdata/influx-cli/v2 v2.2.1-0.20220318222112-88ba3464cd07
+	github.com/influxdata/influxdb v1.9.7
```

Comment:
Think one of your import paths might have changed and caused this to get added to `go.mod`